validate-differential-expression/README.md

```diff
@@ -0,0 +1,88 @@
+For this example workflow, the gene expression datasets and sample metadata are stored in a `data/` directory.
+For more instructions on how to apply this workflow to your own data, see the [section below](#apply-this-workflow-to-your-own-differential-expression-results).
+
+## General requirements and usage
+
+This module requires you to install the following software to run examples yourself:
+
+* [**R**](https://cran.r-project.org/)
+* [**Bioconductor**](https://bioconductor.org/install/)
+* [**tidyverse**](https://www.tidyverse.org/)
+* [**RStudio**](https://www.rstudio.com/products/RStudio/) for working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+
+These requirements can be installed by following the instructions at the links above.
+The example R Notebooks are designed to check if additional required packages are installed and will install them if they are not.
+
+#### RStudio
+
+We have prepared [a quick guide to RStudio](https://github.com/AlexsLemonade/training-modules/blob/master/intro_to_R_tidyverse/00-rstudio_guide.md) as part of our training content that you may find helpful if you're getting started with RStudio for the first time.
+
+Note that the first time you open RStudio, you should select a CRAN mirror.
+You can do so by clicking `Tools` > `Global Options` > `Packages` and selecting a CRAN mirror near you with the `Change` button.
+
+#### Interacting with R Notebooks
+
+You can open an R Notebook by opening the `.Rmd` file in RStudio.
+Note that working with R Notebooks requires certain R packages, but RStudio should prompt you to download them the first time you open one.
+This will allow you to modify and run the R code chunks.
+Chunks that have already been included in an example can be run by clicking the green play button in the top right corner of the chunk or by using **Ctrl + Shift + Enter** (**Cmd + Shift + Enter** on a Mac).
+See [this guide using to R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html#using-notebooks) for more information about inserting and executing code chunks.
+
+#### Obtaining the R Notebooks
+
+To run the example yourself, you will need to [clone](https://help.github.com/articles/cloning-a-repository/) or [download](https://stackoverflow.com/a/6466993) this repository: https://github.com/AlexsLemonade/refinebio-examples
+
+## Apply this workflow to your own differential expression results
+
+If you'd like to adapt an example to include data you've obtained from [refine.bio](https://www.refine.bio/), and your own dataset's differential expression results, we recommend following these steps below.
+
+#### Step 1) Find a refine.bio dataset
+
+Find and download a [refine.bio](https://www.refine.bio/) dataset that has a similar set up and metadata to your own differential expression analysis.
+Use the search bar to look for datasets with key terms.
+Filters can help you find datasets with similar platforms.
+
```

Comment:
Does it make sense to mention Transform options to help them get refine.bio datasets closer to their own dataset? 
Or is that too much for this audience?

Reply:
We decided in an in-person discussion that that is not necessary for this audience.